### ISSUE TITLE
Go live task list

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/docs.asciidoctor.org#27]
This issue tracks the remaining tasks to complete to make the new docs site go live. It's only meant to communicate progress and provide a picture of what tasks remain.

* [x] reconfigure symbolic name for latest version (#26)
* [x] add copy to clipboard button to source blocks (UI) (https://github.com/asciidoctor/asciidoctor-docs-ui/issues/13)
* [x] consolidate explore panel (UI) (https://github.com/asciidoctor/asciidoctor-docs-ui/issues/18)
* [x] allow playbook to control order of components in explore panel (UI) (https://github.com/asciidoctor/asciidoctor-docs-ui/issues/16)
* [x] finalize and merge new topbar (which provides quicklinks to navigate the site) (UI) (https://github.com/asciidoctor/asciidoctor-docs-ui/issues/11)
* [x] reconfigure breadcrumbs to use navtitle for component versions start page (UI) (https://github.com/asciidoctor/asciidoctor-docs-ui/issues/22)
* [x] fix word wrap behavior in tables (apply CSS from Asciidoctor default stylesheet) (UI) (https://github.com/asciidoctor/asciidoctor-docs-ui/issues/27)
* [x] expand nav eagerly (either using embedded script or from template) (https://github.com/asciidoctor/asciidoctor-docs-ui/issues/27)
* [x] only add slot for sidebar TOC if template detects section in page.content (https://github.com/asciidoctor/asciidoctor-docs-ui/issues/32)
* [x] add admonition icons to UI / use icons for menu separator and checklist boxes (UI) (https://github.com/asciidoctor/asciidoctor-docs-ui/issues/26)
* [x] add LICENSE file to this repository
* [x] send PR to asciidoctor repo with asciidoctor component (https://github.com/asciidoctor/asciidoctor/pull/3880)
   * [x] merge docs for asciidoctor component into Asciidoctor repository
   * [x] update playbook with new content source and merge PR (#34)
* [x] set up component in asciidoctor-community-docs for home page (folder: home); move existing docs in that repository into docs folder
* [x] rewrite intro page for site and configure as site start page
* [x] fix table of built-in attributes (most basic fix possible) (https://github.com/asciidoctor/asciidoc-docs/issues/24)
* [x] move intrinsic attributes table to document attributes page (https://github.com/asciidoctor/asciidoc-docs/issues/26)
* [x] set up CHANGELOG file for asciidoc-docs repo (https://github.com/asciidoctor/asciidoc-docs/issues/28)
* [x] remove migration in progress notice on asciidoc-docs repo
* [x] enable checklinks plugin in deploy preview
* [x] fix broken deep links
* [x] add asciidoctor-reveal.js (reveal.js-converter) to docs site (#24)
   * [x] change source branch to 4.1 (#36)
* [x] add asciidoctor-maven-plugin (maven-tools) to docs site
* [x] add asciidoclet to docs site (#25)
  * [x] add asciidoclet to search config (#41)
* [x] add asciidoctorj to docs site (must happen at the same time site is going live)
   * [x] update references to asciidoctorj in asciidoctor core to use index page xref instead of URL to repo (https://github.com/asciidoctor/asciidoctor/pull/3888)
   * [x] add ascidoctorj to search config
* [x] configure Netlify site to use docs.asciidoctor.org (#14)
   * [x] request TLS cert for docs.asciidoctor.org
* [x] add Algolia logo to footer (UI) (https://github.com/asciidoctor/asciidoctor-docs-ui/pull/24)
* [x] enable search for all components (https://github.com/asciidoctor/docs.asciidoctor.org/pull/30)
* [x] add optional "in this project" filter to search (https://github.com/asciidoctor/asciidoctor-docs-ui/issues/37)
* [x] trigger search automatically or on a schedule (#7)

**Comment by mojavelinux** [comment]
We may consider getting the docs running on docs.asciidoctor.org before we start redirecting from asciidoctor.org. This soft launch period will allow us to exercise the site, discover issues, and fix them before making an official public announcement.

**Comment by Mogztter** [comment]
> add Algolia logo to footer

Is it mandatory? There is already a "Search by" mention at the bottom of the search result:

![algolia](https://user-images.githubusercontent.com/333276/101289180-30348780-37fb-11eb-8b3f-69623bdec1ac.png)

> fix table of built-in attributes (most basic fix possible) (asciidoc)

Can I help? What needs to be done?

> enable search for all components

Should I configure something on Asciidoctor.js or Asciidoctor reveal.js?

**Comment by mojavelinux** [comment]
> Is it mandatory (to add an Algolia icon to the footer)

I think it's only fair. They have granted us an OSS plan. And it does as much for the site as Netlify. Plus, we have a space for it.

**Comment by mojavelinux** [comment]
> What needs to be done? (for fix table of built-in attributes).

See https://github.com/asciidoctor/asciidoc-docs/issues/1

**Comment by mojavelinux** [comment]
> Should I configure something on Asciidoctor.js or Asciidoctor reveal.js? (for search)

The search is configured here: https://github.com/asciidoctor/docs.asciidoctor.org/blob/main/docsearch/config.json I would like to see the versions managed automatically from the site build, but if we have to start with manual updates, it's okay.

**Comment by mojavelinux** [comment]
I think we should move the rollout of the redirects to a separate issue since we'll enable it after docs.asciidoctor.org is resolving.

**Comment by graphitefriction** [comment]
@mojavelinux which repository do you want the redirects-rollout checklist/issue to be filed in?

**Comment by mojavelinux** [comment]
The redirects, which we won't put in place until hard launch, are now being tracked here: https://github.com/asciidoctor/asciidoctor.org/issues/981

**Comment by graphitefriction** [comment]
The PR changing `{url-asciidoctorj}` to `xref:asciidoctorj::index.adoc` is ready: https://github.com/asciidoctor/asciidoctor/pull/3888

**Comment by mojavelinux** [comment]
And with that, we're soft launched! :rocket: 

We'll address the official launch and announcement in separate issues.
